CHANGELOG.md

```diff
@@ -4,20 +4,26 @@ This file is used to list changes made in each version of the haproxy cookbook.
 
 ## Unreleased
 
-## 12.1.0 - *2021-06-14*
+## 12.2.0 - _2021-06-21_
+
+- Add `use_promex` property to install.rb to support compiling with Prometheus Exporter support - [@Wicaeed](https://github.com/wicaeed)
```

Comment:
This needs to be moved under `## Unreleased` and the version header deleted as it'll be automatically created upon release